applications/salk-portal/src/components/ExperimentSidebar.jsx

```diff
@@ -126,6 +147,16 @@ const ExperimentSidebar = () => {
   };
 
   const sidebarClass = `${classes.sidebar} scrollbar ${shrink ? `${classes.shrink}` : ``}`;
+  const populationColours = [teal, brown, skyBlue, purple];
+
+  const PopulationLabel = ({labelText, dotColor}) => {
+    return (
+      <Typography variant='span' className='population-label-container' >
+       <Box className='population-label-dot' variant='span' style={{backgroundColor:dotColor}}/>
```

Comment:
@paularah I am pretty sure that material-ui does not give us a variant property on Box yet.